src/Controllers/EditEntryController.php

```diff
@@ -21,9 +21,36 @@ public function __construct($BlogModel)
 
     public function __invoke(Request $request, Response $response, array $args)
     {
-        $EditEntry = $request->getParsedBody();
-        $DbResult = $this->BlogModel->editEntry($EditEntry);
-        return $response->withHeader('location', '/');
+        $ResponseData =
+            [
+                'success' => false,
+                'message' => '',
+                'data' => []
+            ];
+
+        $BlogPost = new Valitron\Validator($_POST);

```

Comment:
so what is this $BlogPost variable? at first I thought it was an object that contained your validation rules but instead it looks like its an object that ends up storing the relevant fields after validation has passed